sdk/core/src/services/github/service.rs

```diff
@@ -1,23 +1,32 @@
+use std::sync::Arc;
+use async_trait::async_trait;
 use crate::services::github::client::{Client as GitHubClient, Repo};
 use crate::services::github::error::Error;
 use platform::git::Service as Git;
+use platform::persistence::mongodb::Store;
 use platform::str::get_random_string;
+use crate::services::github::Commit;
+use platform::persistence::mongodb::Service as MongoService;
+use platform::Error as PlatformError;
+
 
 /// Definition of the GitHubProvider
 #[derive(Debug)]
 pub struct GitHubService {
     org: String,
     client: GitHubClient,
+    pub(crate) store: Arc<Store>
```

Comment:
Pub for tests I assume?

Reply:
Yeah... Maybe but let me see.

Reply:
Nope!  Turns out I didn't need to keep it public.  Great catch